.gitignore

```diff
@@ -138,3 +138,6 @@ dist/
 
 # deploy directory
 deploy
+
+#.q files
+q
```

Comment:
What is this unicode character?

Reply:
That file got somehow auto-generated exactly like that.

Reply:
Can you remove it then?  It makes no sense to me.

Reply:
Will do.